### PR TITLE
update golang-jwt to v4.1.0

### PR DESCRIPTION
**Pull request by NikitaSkrynnik**
Signed-off-by: Nikita Skrynnik <nikita.skrynnik@xored.com>

## Issue link
https://github.com/networkservicemesh/sdk/issues/1055

## How Has This Been Tested?

- [ ] Added unit testing to cover
- [ ] Tested manually
- [ ] Tested by integration testing
- [ ] Have not tested

## Types of changes

- [ ] Bug fix
- [ ] New functionallity
- [ ] Documentation
- [ ] Refactoring
- [ ] CI

### CODE REVIEW

File: pkg/tools/opa/policies/tokens_chained.rego [hunk 1]
```diff
@@ -26,10 +26,14 @@ tokens_chained {
...
 
+contains(arr, elem) {
+  arr[_] = elem
+}
+
 pair_valid(token1, token2) = r { 
 	p1 := payload(token1)
 	p2 := payload(token2)
-	r := p1.aud == p2.sub
+	r := contains(p1.aud, p2.sub)
```
Comment (denis-tingaikin) [inline pkg/tools/opa/policies/tokens_chained.rego:36]: Can we use it inline?
```suggestion
	r := p1.aud[_] = p2.sub
```

Comment (NikitaSkrynnik) [reply pkg/tools/opa/policies/tokens_chained.rego:36]: Yes

**Review by denis-tingaikin — APPROVED** [review]
LGTM

**Comment by edwarnicke** [comment]
LGTM

**Pull request merged**